row.go

```diff
@@ -596,7 +597,7 @@ func (row *Row) loadimpl(dump *dumpfile.Content, initing bool) error {
 	}
 
 	// Set row tag
-	row.tag.Delete(0, row.tag.file.Size(), true)
+	row.tag.Delete(0, row.tag.file.f.Size(), true)
```

Comment:
You wrote a `Size()` forwarding function so you don't need this? It also doesn't matter in this PR because you'll have to fix it up later in any case.